assets/js/components/ClusterDetails/ClusterSettings.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { useParams, useNavigate } from 'react-router-dom';
+
+import { EOS_ARROW_BACK } from 'eos-icons-react';
+import Button from '@components/Button';
+
+function classNames(...classes) {
+  return classes.filter(Boolean).join(' ');
+}
+
+import { Tab } from '@headlessui/react';
+import { ChecksSelection } from './ChecksSelection';
+
+export const ClusterSettings = () => {
+  const { clusterID } = useParams();
+  const navigate = useNavigate();
+
+  const clusters = useSelector((state) => state.clustersList.clusters);
+  const cluster = clusters.find((cluster) => cluster.id === clusterID);
+
+  let [categories] = useState({
```

Comment:
Why do we have these entries in an `useState` function?
They look simple constants to me

Reply:
Copy pasted, going fast and making it work :smile: 

It's gonna change probably.